### PR TITLE
Update DASH PA_VALIDATION table

### PR DESCRIPTION
**Pull request by kperumalbfn**
Update PA_VALIDATION_TABLE information.

### CODE REVIEW

File: doc/dash/dash-sonic-hld.md [hunk 1]
```diff
@@ -674,6 +675,12 @@ key                      = DASH_PA_VALIDATION_TABLE:vni; ENI and VNI as key;
 addresses                = list of addresses used for validating underlay source ip of incoming packets. 
 ```
 
+DASH PA_VALIDATION_TABLE is used only for PL outbound direction. PA address can be either IPV4 or IPV6.
+
+Total PAs per MSEE would be 64 and if there are 64 MSEEs per region, there would be 4K PA_VALIDATION entries.
+
+For more scale numbers, please refer to the [doc]([https://https://github.com/sonic-net/DASH/blob/main/documentation/express-route-service/express-route-gateway-bypass.md])
```
Comment (r12f) [inline doc/dash/dash-sonic-hld.md:682]: the link seems to be broken.

File: doc/dash/dash-sonic-hld.md [hunk 2]
```diff
@@ -674,6 +675,12 @@ key                      = DASH_PA_VALIDATION_TABLE:vni; ENI and VNI as key;
 addresses                = list of addresses used for validating underlay source ip of incoming packets. 
 ```
 
+DASH PA_VALIDATION_TABLE is used only for PL outbound direction. PA address can be either IPV4 or IPV6.
```
Comment (r12f) [inline doc/dash/dash-sonic-hld.md:678]: missing _ for the table name.

File: doc/dash/dash-sonic-hld.md [hunk 3]
```diff
@@ -674,6 +675,12 @@ key                      = DASH_PA_VALIDATION_TABLE:vni; ENI and VNI as key;
...
+
+Total PAs per MSEE would be 64 and if there are 64 MSEEs per region, there would be 4K PA_VALIDATION entries.
```
Comment (r12f) [inline doc/dash/dash-sonic-hld.md:680]: will be better to mention the scale is based on 400G card.

File: doc/dash/dash-sonic-hld.md (hunk 1 above)
Comment (kperumalbfn) [reply doc/dash/dash-sonic-hld.md]: Done

File: doc/dash/dash-sonic-hld.md (hunk 2 above)
Comment (kperumalbfn) [reply doc/dash/dash-sonic-hld.md]: updated

File: doc/dash/dash-sonic-hld.md (hunk 3 above)
Comment (kperumalbfn) [reply doc/dash/dash-sonic-hld.md]: Done

**Review by r12f — APPROVED** [review]

**Pull request merged**